src/plugins/intel_gpu/src/graph/impls/onednn/fully_connected_onednn.cpp

```diff
@@ -402,14 +331,18 @@ struct fully_connected_onednn : typed_primitive_onednn_impl<fully_connected> {
                 }
             }
 
-            if (prim->decompression_zero_point_scalar.has_value() || !prim->decompression_zero_point.empty())
-                zp_mem = prepare_zp_mem(arg, impl_params, is_four_bit_weight, group_size, dzp_data_type, attr);
+            if (!prim->decompression_zero_point.empty()) {
+                auto decompression_zp_idx = !arg.bias_term() ? 3 : 4;
+                auto &zp_node = arg.get_dependency(decompression_zp_idx).as<data>();
+                memory::ptr zp_mem = zp_node.get_attached_memory_ptr();
```

Comment:
where is this used?

Reply:
zp_node and zp_mem are not used. I will remove them. Thanks.